Userland/Applications/Presenter/Presentation.cpp

```diff
@@ -43,12 +43,16 @@ StringView Presentation::author() const
 
 bool Presentation::has_a_next_frame() const
```

Comment:
(note: article is quite unusual in this position, care to remove it from the name of those check methods? i.e. `has_next_frame`/`has_previous_frame`?)